src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
```

Comment:
[Checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 3rd item, use proper names
```suggestion
  for (const action of actions) {
```